src/main/java/envoy/client/ui/Startup.java

```diff
@@ -73,24 +75,24 @@ public void start(Stage stage) throws Exception {
 		EnvoyLog.setFileLevelBarrier(config.getFileLevelBarrier());
 		EnvoyLog.setConsoleLevelBarrier(config.getConsoleLevelBarrier());
 
+		logger.log(Level.INFO, "Envoy was started at " + new Date());
```

Comment:
So, I've recently pulled these changes to try them out and noticed the following: `[2020-06-19 08:26:14] [INFO   ] [envoy.client.ui.Startup] Envoy was started at Fri Jun 19 08:26:14 CEST 2020`. Seems pretty redundant to me, so lets shorten that a bit.

Also, I think the suggested wording depicts the situation a bit clearer because the startup process is not actually completed at the time the log record is published.

```suggestion
		logger.log(Level.INFO, "Envoy starting...";
```